internal/renderers/OpenGL3.go

```diff
@@ -175,9 +177,9 @@ func (renderer *OpenGL3) Render(displaySize [2]float32, framebufferSize [2]float
 				gl.BindTexture(gl.TEXTURE_2D, uint32(cmd.TextureID()))
 				clipRect := cmd.ClipRect()
 				gl.Scissor(int32(clipRect.X), int32(fbHeight)-int32(clipRect.W), int32(clipRect.Z-clipRect.X), int32(clipRect.W-clipRect.Y))
-				gl.DrawElementsWithOffset(gl.TRIANGLES, int32(cmd.ElementCount()), uint32(drawType), indexBufferOffset)
+				gl.DrawElementsBaseVertex(gl.TRIANGLES, int32(cmd.ElementCount()), uint32(drawType), gl.PtrOffset(cmd.IndexOffset()*indexSize), int32(cmd.VertexOffset()))
```

Comment:
`gl.PtrOffset` is deprecated as it will trigger again the `ckeckptr` detector.
Instead, go-gl/glow and go-gl/gl need to be updated with a new overload of the function `DrawElementsBaseVertex` to allow `uintptr` parameters. See their respective readme files for details.

Side-note: At least `go-gl/glow` needs an update. Instead of the `go-gl/gl` dependency, the examples repo could also use its dedicated, specifically generated `gl` bindings.

Please tell how much of all of this you want to do.

Reply:
Unfortunately, I don't have the bandwidth to update multiple repositories - especially repos I've never worked in.

For my own backend, the "abuse" of the pointer type isn't a huge problem

Reply:
Fair enough. I'll leave this PR pending and will see to extend glow & gl on my own.
When ready, I'll take over the PR and finish it. Expect a few days/weeks.

Still, thank you for the contribution! More people could extend the examples in tandem :)

Reply:
👍 

I'll be happy to update this when glow/gl have support for it!
(though it's probably going to be a very simple fix 😄 )